node/service/src/chain_spec.rs

```diff
@@ -312,6 +312,38 @@ fn lagoon_testnet_genesis(
         //5FKuzgFppRcJqs1bYQvrDJ9DrKZaXqrwKggWBk4DyfpXFvoo
         hex!["904e3dea6bcdc6cb523f52cbdedad53c24bbd95692ec690154b0f2c7f0abc55c"].into(),
       ],
+      market_pairs: vec![
+        // ATH_USDC
+        MarketPair {
+          base_asset: assets::Asset::AllTimeHigh.currency_id(),
+          quote_asset: assets::Asset::USDCoin.currency_id(),
+        },
+        // BTC_USDC
+        MarketPair {
+          base_asset: assets::Asset::Bitcoin.currency_id(),
+          quote_asset: assets::Asset::USDCoin.currency_id(),
+        },
+        // ETH_USDC
+        MarketPair {
+          base_asset: assets::Asset::Ethereum.currency_id(),
+          quote_asset: assets::Asset::USDCoin.currency_id(),
+        },
+        // TDFY_BTC
+        MarketPair {
+          base_asset: assets::Asset::Tdfy.currency_id(),
+          quote_asset: assets::Asset::Bitcoin.currency_id(),
+        },
+        // TDFY_ETH
+        MarketPair {
+          base_asset: assets::Asset::Tdfy.currency_id(),
+          quote_asset: assets::Asset::Ethereum.currency_id(),
+        },
+        // TDFY_USDC
+        MarketPair {
+          base_asset: assets::Asset::Tdfy.currency_id(),
+          quote_asset: assets::Asset::USDCoin.currency_id(),
+        },
+      ],
```

Comment:
We're just missing `ETH_BTC` here. It's not currently an active pair but has been in the past and will be again.

Reply:
We can always add/remove market pairs through sudo calls. 
Non-active market pair is currently stored on chain. Any unprocessed swaps will be blocked by this whitelist if they are not supported.